libraries/lib-wave-track/WaveTrack.h

```diff
@@ -1369,6 +1371,17 @@ class WAVE_TRACK_API WaveTrackFactory final
    SampleBlockFactoryPtr mpFactory;
 };
 
+//! TrackAttachments that implement this are notified of changes in the list
+//! of clips
+class WAVE_TRACK_API ClipListListener
+{
+public:
+   virtual ~ClipListListener();
+   //! A clip was added to the list, during deserialization or after
+   virtual void OnNewClip(WaveClip &clip, bool deserializing);
```

Comment:
`deserialization` flag feels like extra parameter

Reply:
I'm not sure of the complete set of messages that will be useful in future.  I could easily use Observer and an enumeration now for creation, deserialization, deletion, ...